opendebates/templates/base.html

```diff
@@ -15,65 +15,45 @@
 <body>
 
 <div class="container">
-  <div class="row white">
-    <div class="login-container pull-right">
+  <header>
+    <div class="header-logos">
+      <img src="{% static "images/badge-democratic-open-forum.png" %}">
+      <img src="{% static "images/logo-progressive-change-institute.png" %}">
+      <img src="{% static "images/logo-cnn.png" %}">
+    </div>
+    <div class="login-container">
       {% if request.user.is_authenticated %}
-      <a href="{# url 'user_profile' #}">
-        {% blocktrans with username=request.user.username %}Welcome, {{ username }}!{% endblocktrans %}
-      </a>
-      <span>||</span>
-      <a href="{% url 'auth_logout' %}">{% blocktrans %}Log out{% endblocktrans %}</a>
-      {% else %}
-      <a href="{% url 'auth_login' %}">{% blocktrans %}Log in{% endblocktrans %}</a> <span> / </span>
-      <a href="{% url 'registration_register' %}">{% blocktrans %}Register{% endblocktrans %}</a>
+        <a href="{# url 'user_profile' #}">
+          {% blocktrans with username=request.user.username %}Welcome, {{ username }}!{% endblocktrans %}
+        </a>
+        <span>/</span>
+        <a href="{% url 'auth_logout' %}">{% blocktrans %}Log out{% endblocktrans %}</a>
+        {% else %}
+        <a href="{% url 'auth_login' %}">{% blocktrans %}Log in{% endblocktrans %}</a> <span> / </span>
+        <a href="{% url 'registration_register' %}">{% blocktrans %}Register{% endblocktrans %}</a>
       {% endif %}
     </div>
-  </div>
-  <div class="row white">
-    <nav class="navbar navbar-default navbar-static-top" role="navigation">
-      <div class="navbar-header">
-
-        <button type="button" class="navbar-toggle" data-toggle="collapse"
-                data-target="#bs-example-navbar-collapse-1">
-          <span class="sr-only">Toggle navigation</span><span class="icon-bar"></span><span
-            class="icon-bar"></span><span class="icon-bar"></span>
-        </button>
+    <div class="header-title">
+      <h1>Welcome to the <br/>Democratic Open Forum</h1>
+      <div class="header-copy">
+        Ask Bernie and Hillary about the issues that are most important to you. Then tune in here to watch the Democratic Open Forum on Thursday, March 3 at 9:00 pm EST. All questions will be chosen from among those that receive the most votes.
```

Comment:
Mark for translation so they can modify this text the way they do the rest of the site?